defi/src/api2/cron-task/appMetadata.ts

```diff
@@ -25,14 +25,12 @@ const protocolChainSetMap: {
 parentProtocols.forEach((protocol: any) => {
   parentProtocolsInfoMap[protocol.id] = protocol
   protocolChainSetMap[protocol.id] = new Set(protocol.chains ?? [])
-  protocol.chainSet = protocolChainSetMap[protocol.id]
   protocol.childProtocols = []
 })
 
 protocols.forEach((protocol: any) => {
   protocolInfoMap[protocol.id] = protocol
   protocolChainSetMap[protocol.id] = new Set(protocol.chains ?? [])
```

Comment:
I am bit surprised that this code works, it relies on data/data1/data2... files containing all the chains related to a protocol, but often we just update the tvl adapter and not this metadata record on the server repo, example: Euler V2 only has "Ethereum" on the metadata, but is also on few other chains